branchtesting/src/main/java/branchexample/App.java

```diff
@@ -1,16 +1,23 @@
 package branchexample;
 
+import java.math.BigDecimal;
+
 public class App 
 {
+	public static String vulnerableString = new String();
+	
     public void aMethod( String[] args )
     {
-    	// TODO fix this
+    	double d = 1.1;
+
+    	BigDecimal bd1 = new BigDecimal(d); // Buggy
     	int i = 0;
     	if (i == 0) {
     		// TODO fix this too..
     		int j = 5 / i;
     	}
-    	System.out.println("IN AMETHOD"); 
+    	System.out.println("IN AMETHOD");
+    	System.out.println("New issue"); 
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/brian-cipollone-sonarsource-gith/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=13&issues=AWWItS6TX2sY87p05UCR&open=AWWItS6TX2sY87p05UCR)

[ISSUE_KEY]: AWWItS6TX2sY87p05UCR